src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -157,6 +152,7 @@ static int32_t draw_dispatch(lv_draw_unit_t * draw_unit, lv_layer_t * layer)
 
     /* Return 0 is no selection, some tasks can be supported by other units. */
     if(!t || t->preferred_draw_unit_id != VG_LITE_DRAW_UNIT_ID) {
+        lv_vg_lite_finish(draw_unit);
         return -1;
     }
```

Comment:
@kisvegabor 
If there are no more drawing tasks, it will wait for the GPU drawing to complete and clear flush_count.